src/OrchardCore.Modules/OrchardCore.Tenants/Controllers/AdminController.cs

```diff
@@ -167,11 +176,19 @@ public async Task<IActionResult> Index(TenantIndexOptions options, PagerParamete
             };
 
             // We populate the SelectLists
+            model.Options.TenantsCategories = allSettings
+                .GroupBy(t => t["Category"])
+                .Where(t => !String.IsNullOrEmpty(t.Key))
+                .Select(t => new SelectListItem(t.Key, t.Key))
+                .ToList();
+
+            model.Options.TenantsCategories.Insert(0, new SelectListItem(S["All"], DefaultCategory, selected: true));
```

Comment:
This is only selected:true when it’s all or empty isn’t it?

so here and in the above you need to do a comparison against the currently selected value to set the is selected property

Reply:
I think no need to ignore the comparison casing here because the `DefaultCategory` is not editable. Regarding the check you are right need to be changed